modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
+
+* With this update, when you delete task runs or pipeline runs, you can use the `--parent-resource` and `--keep-since` options together. For example, the `tkn pipelinerun delete --pipeline pipelinename --keep-since 5` command preserves pipeline runs whose parent resource is named `pipelinename` and whose age is five minutes or less. The `$ tkn tr delete -t taskname --keep-since 5` and `$ tkn tr delete --clustertask taskname --keep-since 5` commands work similarly for task runs. (link:https://github.com/tektoncd/cli/pull/1443[#1443])
+
+* This update adds support for triggers resources to work with `v1beta1` resources (link:https://github.com/tektoncd/cli/pull/1446[#1446], link:https://github.com/tektoncd/cli/pull/1449[#1449], link:https://github.com/tektoncd/cli/pull/1450[#1450], link:https://github.com/tektoncd/cli/pull/1454[#1454], link:https://github.com/tektoncd/cli/pull/1455[#1455])
+
+* This update adds an `ignore-running` flag to the `tkn pipelinerun delete` and `tkn taskrun delete` commands. (link:https://github.com/tektoncd/cli/pull/1445[#1445])
+
+* This update adds a `create` subcommand to the `tkn task` and `tkn clustertask` commands. (link:https://github.com/tektoncd/cli/pull/1359[#1359])
+
+=== CLI 0.20.0 release
+
+* With this update, if you use the `tkn pipelinerun delete --all` command, you can preserve runs that are younger than a specified number of minutes by using the new `--keep-since <minutes>` option. For example, to keep pipeline runs that are less than five minutes old, you enter `$ tkn pipelinerun delete -all --keep-since 5`. (link:https://github.com/tektoncd/cli/pull/1392[#1392])
+
+* With this update, when you use the `tkn pipelinerun delete --all` command, you can use the new `--label <string>` option to filter the pipeline runs by label. Optionally, you can use the `--label` option with `=` and `==` as equality operators, or `!=` as an inequality operator. For example, the `tkn pipelinerun delete --all --label asdf` and  `tkn pipelinerun delete --all --label==asdf` commands both delete all the pipeline runs that have the `asdf` label. (link:https://github.com/tektoncd/cli/pull/1402[#1402])
+
+* With this update, you can fetch the version of installed Tekton components from the `ConfigMap` or, if the `ConfigMap` is not present, from the deployment controller. (link:https://github.com/tektoncd/cli/pull/1393[#1393])
+
+=== Tekton Triggers v0.16.0
+
+* With this update, triggers support the `configmap feature-flags` and `configmap config-defaults` commands to configure feature flags and to set default values, and adds `serviceAccountName` to the defaults. (link:https://github.com/tektoncd/cli/pull/1110[#1110], link:https://github.com/tektoncd/cli/pull/1182[#1182])
```

Comment:
```suggestion
* With this update, triggers support the `oc configmap feature-flags` and `oc configmap config-defaults` commands to configure feature flags and to set default values, and adds `serviceAccountName` to the defaults. (link:https://github.com/tektoncd/cli/pull/1110[#1110], link:https://github.com/tektoncd/cli/pull/1182[#1182])
```
seems like an incomplete command, should they have `oc` or `tkn` before `configmap`?
Also, the "and adds `serviceAccountName` to the defaults." bit seems like it could be its own bullet, especially if they have two separate github issues / PRs?

Reply:
* With this update, triggers support the `feature-flags` and `config-defaults` configmap to configure feature flags and to set default values respectively. (link:https://github.com/tektoncd/cli/pull/1110[#1110], link:https://github.com/tektoncd/cli/pull/1182[#1182])

How about this?

Reply:
Updated. Thanks.

Reply:
We need to change links 
Its pointing to `cli`

correct links
https://github.com/tektoncd/triggers/pull/1182
https://github.com/tektoncd/triggers/pull/1110

Reply:
@savitaashture makes sense to me

Reply:
Updated. Thanks.